samples/radio_test_bm/src/nrf_wifi_radio_test_shell.c

```diff
@@ -1789,8 +1834,12 @@ SHELL_STATIC_SUBCMD_SET_CREATE(
                   "           For 106 ru_tone:  1 to 2\n"
                   "           For 242 ru_tone:  1",
                   RTSH(nrf_wifi_radio_test_set_ru_index), 2, 0),
-    SHELL_CMD_ARG(init, NULL, "<val> - Primary channel number",
+    SHELL_CMD_ARG(rt_init, NULL, "<val> - Primary channel number",
```

Comment:
I can drop the rename and instead add nrf70_init and nrf70_deinit APIs? and also may be leave in the automatic init too? This makes it completely backwards compatible and also gives the option for user to deinit and init nrf70?